src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -480,30 +473,30 @@ def utask_main(uworker_input):
     # We didn't crash at all. This might be a legitimately unreproducible
     # test case, so it will get marked as such after being retried on other
     # bots.
-    testcase = data_handler.get_testcase_by_id(uworker_input.testcase_id)
+    testcase = data_handler.get_testcase_by_id(testcase_id)
```

Comment:
Can you just reuse the testcase? You won't be able to call get_testcase_by_id from utask_main once we migrate.

Reply:
In this case you should probably return a specific new kind of error and then handle it in utask_postprocess. I won't leave these comments everywhere since this is just an initial PR.

Reply:
ahh yeah we talked about this in the meeting last week right - updated most but concerned about
"# We can't call check_deadline_exceeded_and_store_partial_minimized_testcase
    # at this point because we do not have a test case to store."

why dont we have a testcase to store at that point? (raised it as a separate comment so you dont have to go digging for it)

Reply:
jk i am unable to add a comment there - its line 515!